### PR TITLE
Expose the line number in the parsed content

### PR DESCRIPTION
**Pull request by tosky**
The information about the current line number is already available,
just expose it.

### CODE REVIEW

File: iniparse/ini.py [hunk 1]
```diff
@@ -282,10 +282,15 @@ def set_value(self, data):
             else:
                 self.add(EmptyLine())
 
+    def get_line_number(self):
+       return self.contents[0].line_number if self.contents else None
```
Comment (cnsnyder) [inline iniparse/ini.py:286]: Will contents always be an array and will the item at the first index always have a line number attribute?

Comment (tosky) [reply iniparse/ini.py:286]: It is defined above as self.contents = [], so it is always an array.
If I read the code correctly, contents is populated through the `add` method whose argument seems to always be a child class of LineType.

**Comment by cnsnyder** [comment]
This looks good to me! Thanks for the contribution. Would you mind rebasing on candlepin:main so we can get the tests to run via packit as well prior to merge?

**Comment by tosky** [comment]
Sure, rebase incoming!

**Pull request merged**